packages/eslint/src/generators/init/init.spec.ts

```diff
@@ -73,6 +83,30 @@ describe('@nx/eslint:init', () => {
     `);
   });
 
+  it('should add the eslint extension to the recommended property', () => {
+    writeJson(tree, '.vscode/extensions.json', {
+      recommendations: [
+        'nrwl.angular-console',
+        'angular.ng-template',
+        'esbenp.prettier-vscode',
+      ],
+    });
+
+    updateVsCodeRecommendedExtensions(tree);
```

Comment:
Let's test the extension is added when invoking the `lintInitGenerator`. That would guard against any potential refactor that, by mistake, could remove the invocation of the `updateVsCodeRecommendedExtensions` helper.
```suggestion
  it('should add the eslint extension to the recommended property', async () => {
    writeJson(tree, '.vscode/extensions.json', {
      recommendations: [
        'nrwl.angular-console',
        'angular.ng-template',
        'esbenp.prettier-vscode',
      ],
    });

    await lintInitGenerator(tree, options);
  });
```

Reply:
Ok I would love to save code that check result after lintInitGenerator executed